airflow-core/src/airflow/models/taskinstance.py

```diff
@@ -392,7 +392,7 @@ def set_current_context(context: Context) -> Generator[Context, None, None]:
             )
 
 
-def _stop_remaining_tasks(*, task_instance: TaskInstance, session: Session):
+def _stop_remaining_tasks(*, task_instance: TaskInstance, task_teardown_map=None, session: Session):
```

Comment:
While we are in here, we should record an "audit log" against the TIs we stop so it shows up in the UI, otherwise it's a bit confusing that the state is failed but there is no task log etc.

Reply:
Handled it in 25abe363862861ababb4ee7c38c6b2dc52ee7df2